Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -50,7 +50,8 @@
   public static final int OVERLAY_WIIMOTE_SIDEWAYS = 2;
   public static final int OVERLAY_WIIMOTE_NUNCHUK = 3;
   public static final int OVERLAY_WIIMOTE_CLASSIC = 4;
-  public static final int OVERLAY_NONE = 5;
+  public static final int OVERLAY_WIIMOTE_DRUM = 5;
+  public static final int OVERLAY_NONE = 6;
```

Comment:
The value of `OVERLAY_NONE` should remain `5`, otherwise you'll break existing configs.